README.md

```diff
@@ -1,5 +1,10 @@
 # Paragon
 
+## Warning: Package Moved
+
+This package will no longer receive updates and the latest version can be found
+at https://www.npmjs.com/package/@openedx/paragon
```

Comment:
[clarification] Visiting this link brings me to a login screen. I don't think we should merge this until the NPM url for `@openedx/paragon` is broadly available to unauthenticated users.

Reply:
@adamstankiewicz the URL breaks because we haven't published it to the new scope yet, that will be in a follow up PR that will land right after this one so the time when this URL is unavailable will be a few minutes at most.

Reply:
Thanks for the added context!